ui/apps/everest/src/pages/database-form/database-form-body/steps/first/first-step.tsx

```diff
@@ -244,7 +249,7 @@ export const FirstStep = ({ loadingDefaultsForEdition }: StepProps) => {
           name={DbWizardFormFields.k8sNamespace}
           label={Messages.labels.k8sNamespace}
           loading={namespacesFetching}
-          options={namespaces || []}
+          options={permittedNamespaces || []}
```

Comment:
should we show db in dblists only for permittedNamespaces?

Reply:
The read permission for the dblist table is already handled by the BE. Here we limit the namespaces options to only those who have create permissions